lib/src/components/NxNexusPageHeader/NxNexusPageHeader.tsx

```diff
@@ -28,10 +26,12 @@ function HeaderLink({ name, href, current }: HeaderLinkProps) {
 // visible for testing
 export { HeaderLinkProps, HeaderLink };
 
-export default function NxNexusPageHeader({ links, homeLink, productInfo, children }: Props) {
-  const logoEl = <img src={logoImg} className="nx-product__logo-image" alt="⬡ Sonatype"/>,
+export default function NxNexusPageHeader({ links, homeLink, productInfo, children, imgURL }: Props) {
+  const logoImg = imgURL || '../../assets/img/logo_nexus_generic.svg',
```

Comment:
You still need to use webpack `require` to get the path - that way it'll be aware of the placement of the icon within the consuming app's build output, rather than assuming it matches RSC's source tree directory structure which would be very unlikely.

Reply:
> You still need to use webpack `require` to get the path - that way it'll be aware of the placement of the icon within the consuming app's build output, rather than assuming it matches RSC's source tree directory structure which would be very unlikely.

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/3637f962ab6826c8fbdd3eac0762aea9b6aa98b7

Reply:
> Ok: 3637f96

Since this is a file import I would strongly prefer it be at the top level of the file.

Reply:
> > Ok: [3637f96](https://github.com/sonatype/sonatype-react-shared-components/commit/3637f962ab6826c8fbdd3eac0762aea9b6aa98b7)
> 
> Since this is a file import I would strongly prefer it be at the top level of the file.

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/156df1fb89516f53c9f7b74cc9d6b15440c5c072